src/elements/content-sidebar/activity-feed/comment/__tests__/RepliesToggle.test.js

```diff
@@ -0,0 +1,82 @@
+// @flow
+
+import React from 'react';
+import { IntlProvider } from 'react-intl';
+import { fireEvent, render, screen } from '@testing-library/react';
+import RepliesToggle from '../RepliesToggle';
+
+jest.mock('react-intl', () => ({
+    ...jest.requireActual('react-intl'),
```

Comment:
Why do we need to require the actual implementation while just mocking it here? Were you seeing errors by not doing this?

Reply:
Yeah seems like it just can't find any of the messages without this. I believe its throughout the sidebar

Reply:
Ok. This is used elsewhere in the code. Will reach out to Wiola and find out why.